src/allotropy/parsers/moldev_softmax_pro/softmax_pro_structure.py

```diff
@@ -189,20 +170,18 @@ def is_empty(self) -> bool:
         return not self.dimensions or not self.values
 
 
-@dataclass(frozen=True)
-class PlateBlock(Block):
-    block_type: str
-    name: Optional[str]
-    export_format: Optional[str]
-    read_type: Optional[str]
-    data_type: Optional[str]
+@dataclass
```

Comment:
I don't believe I see anywhere where `PlateHeader` is modified, why did we remove `frozen=True`?